AndelaOSP/Andela-Socials
# #159075054 Implement navigation bar, the upper part

#### What Does This PR Do?

Create basic Navigation bar features including search, notification, profile and dropdown

#### Description Of Task To Be Completed

- UI implementation of the upper part of the NavBar as this [mockup](https://www.figma.com/file/Yn3JRZ3YLBVSg4o8L9dhIAv2/Andela_Socials?node-id=783%3A302) shows

#### How can this be manually tested?

Assuming you have the app locally setup, if not follow this [README.md](https://github.com/AndelaOSP/Andela-Socials/blob/develop/README.md) to do so.

- Ensure necessary dependencies are installed and to start the app run `scripts/start.sh` 
- Navigate to `localhost:9000/home`, you should be able to see a Navigation Bar that looks like the on e in the screenshot below.
- If you click on the `Bell` or `Profile` button, you should be able to see a DropDown

#### What are the relevant Pivotal Tracker stories?

[159075054](https://www.pivotaltracker.com/n/projects/2174978/stories/159075054)

#### Screenshot(s)

<img width="1440" alt="screen shot 2018-07-30 at 11 57 24" src="https://user-images.githubusercontent.com/19430799/43388369-902c97dc-93f1-11e8-9238-76c007d60390.png">
